ansible/roles/cuda/tasks/main.yaml

```diff
@@ -25,20 +25,28 @@
   register: cuda__dash_case_cuda_version
   changed_when: false
 
-- name: Install CUDA libraries except for cuda-drivers
+- name: Install CUDA devel libraries except for cuda-drivers
   become: true
   ansible.builtin.apt:
     name:
-      - cuda-cudart-dev-{{ cuda__dash_case_cuda_version.stdout }}
       - cuda-command-line-tools-{{ cuda__dash_case_cuda_version.stdout }}
       - cuda-minimal-build-{{ cuda__dash_case_cuda_version.stdout }}
-      - cuda-libraries-dev-{{ cuda__dash_case_cuda_version.stdout }}
-      - cuda-nvml-dev-{{ cuda__dash_case_cuda_version.stdout }}
-      - libnpp-dev-{{ cuda__dash_case_cuda_version.stdout }}
       - libcusparse-dev-{{ cuda__dash_case_cuda_version.stdout }}
       - libcublas-dev-{{ cuda__dash_case_cuda_version.stdout }}
-      - libnccl-dev
+      - libcurand-dev-{{ cuda__dash_case_cuda_version.stdout }}
```

Comment:
Since I could not find any official cuda and tensorRT runtime-only packages available, I tried to find optimum runtime dependencies and tested the Autoware with rosbag-replay simulator. But any suggestion could help to make it better :)

Reply:
thanks
(I can't resolve conversations)